docs/tutorials/pre_executed/timeseries.ipynb

```diff
@@ -0,0 +1,1471 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        return {"points": len(pts)}
</pre><p><span class="mention" data-value="hombit" data-denotation-char="@">﻿<span contenteditable="false">@hombit</span>﻿</span> I remember our discussion from last week about this being slower than count_nested. Will work on changing this.</p><p> </p><p>The other question might be whether <code>.reduce</code> or <code>.map_partitions</code> is better for this, and whether it matters.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
After a lot of trying, I found the neat workaround with `.map_partitions` that I describe in https://github.com/lincc-frameworks/nested-pandas/issues/291, and when that issue is complete, we won't even need the tiny workaround, at least for this use case.